### PR TITLE
OpenXR stereo improved

### PR DESCRIPTION
**Pull request by lvonasek**
Using manual slider to adjust stereo separation seems to be too complicated for the users. In this PR I use the extracted configuration from PPSSPP PCVR version from 2016 to get the world scale and use this world scale for 6DoF and stereo separation.

### CODE REVIEW

File: Core/Compatibility.cpp [hunk 1]
```diff
@@ -111,3 +120,9 @@ void Compatibility::CheckSetting(IniFile &iniFile, const std::string &gameID, co
 		*flag |= all;
 	}
 }
+
+void Compatibility::CheckSetting(IniFile &iniFile, const std::string &gameID, const char *option, float *flag) {
+	std::string value;
+	iniFile.Get(option, gameID.c_str(), &value, "0");
+	*flag = stof(value);
```
Comment (hrydgard) [inline Core/Compatibility.cpp:127]: don't need `stof`, you can just get a float directly from the inifile without going through a string.

File: assets/compatvr.ini [hunk 2]
```diff
@@ -0,0 +1,620 @@
+# ========================================================================================
+# compatvr.ini for PPSSPP
+# ========================================================================================
+#
+# This file is not meant to be user-editable, although is kept as a separate ini
```
Comment (hrydgard) [inline assets/compatvr.ini:5]: Just add something specific about VR here.

File: Core/Compatibility.cpp (hunk 1 above)
Comment (lvonasek) [reply Core/Compatibility.cpp:127]: `class Section` has a method to parse floats, `class IniFile` doesn't...

Comment (hrydgard) [reply Core/Compatibility.cpp:127]: hm, fair enough. Feel free to add it though!

**Review by hrydgard — APPROVED** [review]

**Comment by lvonasek** [comment]
Could you merge this? I do not have rights to merge it.

File: Common/VR/PPSSPPVR.cpp [hunk 3]
```diff
@@ -285,9 +285,19 @@ void UpdateVRProjection(float* projMatrix, float* leftEye, float* rightEye) {
...
+	} else {
+		VR_SetConfig(VR_CONFIG_6DOF_PRECISE, false);
+	}
+	VR_SetConfig(VR_CONFIG_6DOF_SCALE, (int)(scale * 1000000));
 }
 
-void UpdateVRView(float* projMatrix, float* leftEye, float* rightEye) {
-	VR_TweakView(leftEye, projMatrix, VR_VIEW_MATRIX_LEFT_EYE);
-	VR_TweakView(rightEye, projMatrix, VR_VIEW_MATRIX_RIGHT_EYE);
+void UpdateVRView(float* leftEye, float* rightEye) {
```
Comment (unknownbrackets) [inline Common/VR/PPSSPPVR.cpp:300]: nit: We prefer putting the * against the variable not the type.  Consider: `float *ptrToFloat, notAPointer;`.  It'd be nice if this wasn't how C++ worked, but pretending it works in a more sensible way in our syntax won't make it true.

-[Unknown]

**Pull request merged**